net-wireless/hostapd/hostapd-2.9-r3.ebuild

```diff
@@ -99,9 +99,6 @@ src_configure() {
 	echo "CONFIG_EAP=y" >> ${CONFIG}
 	echo "CONFIG_ERP=y" >> ${CONFIG}
 	echo "CONFIG_EAP_MD5=y" >> ${CONFIG}
```

Comment:
you're doing I/O with external commands, this always needs a `|| die`. Instead of `echo`ing, I suggest you use the `cat <` trick

Reply:
The whole ebuild has `echo CONFIG_BLAH` without `|| die`. Should I do the modification everywhere then?

Reply:
yes please, but I'll do this on merge